src/encoded/types/experiment.py

```diff
@@ -336,6 +337,28 @@ def perturbed(self, request, replicates=None):
             return any(bio_perturbed)
         return False
 
+    @calculated_property(schema={
+        "title": "[Deprecated] Analysis",
```

Comment:
it's not critical, but the language here is strange to me.  "Deprecated" to me means "currently bad".   It seems like you mean it here as 'we will change the name of this property in the future'